cranelift/codegen/src/ir/dfg.rs

```diff
@@ -167,6 +167,11 @@ impl DataFlowGraph {
         self.blocks.is_valid(block)
     }
 
+    /// Make a BlockWithArgs, bundling together the block and its arguments.
+    pub fn block_with_args(&mut self, block: Block, args: &[Value]) -> BlockCall {
```

Comment:
Do you suppose this function should be renamed to `block_call` now, or perhaps `make_block_call` or something? I think calling it `block_with_args` isn't terrible but it's worth taking a moment to think about it.

Also, there are various comments that still refer to `BlockWithArgs`, like this one and a bunch of places in `instructions.rs`.